loader.go

```diff
@@ -387,6 +387,8 @@ const suiteDetailedSchema = `
                   "type": "object",
                   "minProperties": 1
                 },
+				"newBody": {},
```

Comment:
what if both exact and non-exact bodies are provided in the same suite
that does not seem a real use case we want to support
either validation or bool property instead of exactBody is needed